docs/USAGE-GUIDE.md

```diff
@@ -493,6 +493,18 @@ int s2n_config_free(struct s2n_config *config);
 
 **s2n_config_free** frees the memory associated with an **s2n_config** object.
 
+### s2n\_config\_set\_security\_policy
+
+```c
+int s2n_config_set_versioned_security_policy(struct s2n_config *config,
+                                      const char *version);
+```
+
+**s2n_config_set_versioned_security_policy** S2N makes opinionated decisions about which ciphers,
+key encapsualtion mechanisms, signature algorithms, and curves can be used together. These
+combinations are called *security policies*. This function is simply a wrapper for
+*s2n_config_set_cipher_preferences*. The name better describes the behavior behind the scenes.
+
 ### s2n\_config\_set\_cipher\_preferences
```

Comment:
We should update the documentation for `s2n_config_set_cipher_preferences` to indicate that it is `S2N_DEPRECATED` API.

Reply:
updated